aggregator/.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
The `aggregator` folder should not contain `.github` or `.mvn` subfolders.

Reply:
I know :-) it wasn't ready for review. Had to open a new PR to check if the submodule stuff works.

Reply:
Oh sorry, I thought the PR was ready since it's not a draft 😃